app/models/text_field_type.rb

```diff
@@ -3,7 +3,9 @@ class TextFieldType < FieldType
 
   validates :text, presence: true, if: :validate_presence?
   validate :text_length, if: :validate_length?
-
+  validate :less_than, if: :validate_min?
```

Comment:
`TextFieldType` should no longer contain numeric-related functionality, so we can wipe away changes to this file.